config/scorecard/kuttl/00-installation/02-cluster-storage.yaml

```diff
@@ -3,5 +3,4 @@ kind: RHMI
 metadata:
   name: rhoam
 spec:
-  useClusterStorage: 'true'
-  selfSignedCerts: true
```

Comment:
removing `selfSignedCerts: true`, since this was a temporary workaround for a demo